packages/cozy-harvest-lib/src/components/TriggerManager.jsx

```diff
@@ -444,7 +446,9 @@ DumbTriggerManager.propTypes = {
   flow: PropTypes.object,
   flowState: PropTypes.object,
   // Used to inject a component around OAuthForm, and so customize the UI from the app
-  OAuthFormWrapperComp: PropTypes.node
+  OAuthFormWrapperComp: PropTypes.node,
+  // custom intents api. Can have fetchSessionCode, showInAppBrowser, closeInAppBrowser at the moment
+  intentsApi: PropTypes.object
```

Comment:
Since we pass `intentsApi` several times you can: 
```
export const IntentsApi = PropTypes.shape({
  fetchSessionCode: PropTypes.func,
  showInAppBrowser: PropTypes.func,
  closeInAppBrowser: PropTypes.func
})
```

in a proptypes.js file. And then import it everytime you need to check the intentsApi. 

Having `object` is not enough. It's like having `any` it doesn't give us any information 🤷‍♂️